backend/app/Repositories/SessionRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\Repositories\Contracts\SessionRepository as SessionRepositoryInterface;
+use Illuminate\Support\Collection;
+
+class SessionRepository implements SessionRepositoryInterface
```

Comment:
repository implementation should describe implementation itself, in our case it should be `EloquentSessionRepository`